src/index.html

```diff
@@ -1,15 +1,228 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label
+          for="surname"
+          class="fieldset_label"
+        >
+          Surname:
+          <input type="text"
+                 id="surname"
+                 name="surname"
+                 required>
+        </label>
+        <label
+          for="name"
+          class="fieldset_label"
+        >
+          Name:
+          <input
+            type="text"
+            id="name"
+                 name="surname" required
+          >
+        </label>
+        <label
+          for="age"
+          class="fieldset_label"
+        >
+          How old are you?
+          <input
+            type="number"
+            id="age"
+            name="age"
+            required
+            value="12" min="1"
+            max="100"
+          >
```

Comment:
1 attribute = 1 line

```suggestion
          <input
            type="number"
            id="age"
            name="age"
            required
            value="12"
            min="1"
            max="100"
          >
```